mocap_optitrack_driver/include/mocap_optitrack_driver/mocap_optitrack_driver.hpp

```diff
@@ -84,21 +84,22 @@ class OptitrackDriverNode : public mocap_control::ControlledLifecycleNode
   void initParameters();
 
   void process_frame(sFrameOfMocapData * data);
+  NatNetClient * client;
```

Comment:
Why you made this member public?

Reply:
Left-over from a previous commit, when I didn't understand the NatNetClient properly, yet.